src/styles/_reset.scss

```diff
@@ -169,3 +169,21 @@ a {
     color: inherit;
   }
 }
+
+article > a {
+  color: #3078df;
```

Comment:
1. The colors you use here do not really much design specs (https://drive.google.com/open?id=0B3y3SEA9cIEkRWt2TVpXMDhQbTA)!
2. Also, you should not put color codes directly to your custom styling. We have all proper colors defined as variables here: https://github.com/topcoder-platform/community-app/blob/develop/src/styles/_variables.scss#L43
So, you just check what is the variable name for that color you need, and put you scss `color: $tc-black;` or whatever you need.